src/utils/__tests__/kebabCase-test.js

```diff
@@ -0,0 +1,19 @@
+import kebab from "../kebabCase";
+
+test("converts common cases", () => {
+  expect(kebab("foobar")).toEqual("foobar");
+  expect(kebab("fooBar")).toEqual("foo-bar");
+  expect(kebab("foo_bar")).toEqual("foo-bar");
+  expect(kebab("FooBar")).toEqual("foo-bar");
+  expect(kebab("fooBar")).toEqual("foo-bar");
+  expect(kebab("foo bar")).toEqual("foo-bar");
+  expect(kebab("FOO BAR")).toEqual("foo-bar");
+  expect(kebab("foo-bar")).toEqual("foo-bar");
```

Comment:
You can use `.toBe` when you're asserting on literal strings.

Reply:
I'm afraid beacause `toBe` is kinda strict. I'll change it. :)